OpenLIFUHome/OpenLIFUHome.py

```diff
@@ -24,6 +18,8 @@
     vtkMRMLTransformNode,
 )
 
+import OpenLIFULib
+
 if TYPE_CHECKING:
     import openlifu # This import is deferred to later runtime, but it is done here for IDE and static analysis purposes
     import openlifu.db
```

Comment:
It will also be nice if you can add a copy of these lines to the top of OpenLIFULib, for the sake of the type checker

Reply:
(I guess you don't need the `openlifu.db` part in `OpenLIFULib`)